.github/macosdependency.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Why not add this in `.github/manimdependency.json` instead of creating a new file.